src/test/scala/org/scalameter/MethodInvocationCountTest.scala

```diff
@@ -0,0 +1,7 @@
+package org.scalameter
+
+class MethodInvocationCountTest extends InvocationCountMeasurerTest {
+  test("MethodInvocationCountTest.allocations should be deterministic") {
+    checkInvocationCountMeasurerTest(new org.scalameter.examples.MethodInvocationCountTest1)
```

Comment:
I think that the problem arises with the lookup table key. It must be in the context before invocation count `measure` method and it is created after writing instrumented classes. So either `Instrumentation.writeInstrumentedClasses` must be refactored to do some in-memory check (but I think that this could be inefficient for more computationally expensive matchers, like the on-way `Descendants` class matcher), or it should insert `methodInvocationLookupTable` into the context before calling measurer `measure` (so in the executor or in performance test (?))
Or do you see other alternatives?